server/helpers/authHelpers.js

```diff
@@ -1,44 +1,53 @@
 import jwt from 'jsonwebtoken';
 import { config } from 'dotenv';
 import sgMail from '@sendgrid/mail';
+import bcrypt from 'bcrypt';
 import { User } from '../sequelize/models/index';
 
+
 config();
 sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
 /**
  * This class contains all methods
- * required to handle
- * authentication data to the database.
+ * required to handle all
+ * password reset and update.
```

Comment:
Consider updating this comment to fit the current use. It is not only about the password. Use a comment that is general in this case to cover each use.

Reply:
updated

Reply:
Not yet, Double-check. The comment has not changed. This class is handling several things including sending emails, generating, and verifying tokens. Thus, find an appropriate comment that encompasses the functionalities.

Reply:
check again to see if it is okay now